src/tools/rbd_mirror/instance_watcher/Types.h

```diff
@@ -44,7 +44,7 @@ struct ImagePayloadBase : public PayloadBase {
   std::string peer_mirror_uuid;
   std::string peer_image_id;
 
-  ImagePayloadBase() : PayloadBase() {
+  ImagePayloadBase() : PayloadBase(), global_image_id{""}, peer_mirror_uuid{""}, peer_image_id{""} {
```

Comment:
@trociny Changed the code locally as you suggested and the warnings came back during the build.

`ImagePayloadBase() {
}
`

and 

`  SyncPayloadBase() {
  }
`